web/sites/default/config/default/views.view.affinity_groups_with_tag.yml

```diff
@@ -918,7 +918,7 @@ display:
           element_wrapper_type: ''
           element_wrapper_class: ''
           element_default_classes: true
-          empty: '<a href="/user/login" class="btn btn-sm btn-primary"><small>Login to join</small></a>'
+          empty: '<a href="/user/login?destination=/affinity-groups/access-support" class="btn btn-sm btn-primary"><small>Login to join</small></a>'
```

Comment:
We need to figure out how to get the destination dynamically. We don't always want to go to the access-support Affinity Group.